blog/2022-11-03-short-circuiting.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Array short-circuiting
+author: Philip Munksgaard
+description: In-place updates without overhead
+---
+
+# Situation
+
+Futhark is a parallel high-level array-oriented language.  Parallelism is explicitly
```

Comment:
You should introduce yourself so they know it's not just me as usual!

Reply:
Good point.